root_s3_object_lifecycle.tf

```diff
@@ -39,4 +42,22 @@ locals {
         noncurrent_days = local.default_expiration_days
       }
   }]
+
+  non_prod_export_bucket_lifecycle_rules = [
```

Comment:
I would rename this rule: `export_bucket_lifecycle_rules` and make it generic for all environments. See how the `dirty_bucket_lifecycle_rules` is configured